spec/index.bs

```diff
@@ -4411,11 +4418,11 @@ typedef double GPUPipelineConstantValue; // May represent WGSL's bool, f32, i32,
                 :: If |entry|.{{GPUBindGroupLayoutEntry/buffer}}.{{GPUBufferBindingLayout/type}} is:
                     <dl class=switch>
                         : {{GPUBufferBindingType/"uniform"}}
-                        :: The |binding| is a uniform buffer.
+                        :: |binding| is declared with the storage class `uniform`.
```

Comment:
We should clarify that `binding` refers to a global variable. Maybe we can rename it to `variable`?